src/backend/mscoffobj.c

```diff
@@ -708,7 +725,8 @@ void MsCoffObj::term(const char *objfilename)
         return;
 #endif
 
-    build_syment_table();
+    bool bigobj = scnhdr_cnt > 0xffff;
```

Comment:
Found this reference http://llvm.org/docs/doxygen/html/Support_2COFF_8h_source.html saying the limit is 65279.